src/CastsEnums.php

```diff
@@ -68,6 +68,16 @@ public function setAttribute($key, $value)
         parent::setAttribute($key, $value);
     }
 
+    /**
+     * Convert the model's attributes to an array.
+     *
+     * @return array
+     */
+    public function attributesToArray()
```

Comment:
There's one question that tackles my mind: will it not break the base method with the same name?
https://github.com/laravel/framework/blob/5.6/src/Illuminate/Database/Eloquent/Concerns/HasAttributes.php#L85

Shouldn't it also call `parent::attributesToArray()`?